### PR TITLE
TURTLES-713 Properly configure agent.plugin timeout value

### PR DESCRIPTION
**Pull request by npawelek**
Properly configure agent.plugin timeout value for all plugin templates.

### CODE REVIEW

File: playbooks/files/rax-maas/plugins/swift-recon.py [hunk 1]
```diff
@@ -432,6 +432,9 @@ def make_parser():
     parser.add_argument('--swift-recon-path',
                         default='/usr/local/bin',
                         help='The path for the swift-recon directory.')
+    parser.add_argument('-t',
+                        default='30',
```
Comment (BjoernT) [inline playbooks/files/rax-maas/plugins/swift-recon.py]: Please spell out the units like seconds or milli seconds etc.

File: playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2 [hunk 2]
```diff
@@ -16,6 +16,7 @@ disabled    : "{{ (inventory_hostname != groups['mons'][0] or check_name | match
 details     :
     file    : run_plugin_in_venv.sh
     args    : {{ ceph_args }}
+    timeout : {{ (maas_check_timeout_override[label] | default(maas_check_timeout) * 1000) }}
```
Comment (BjoernT) [inline playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2]: Lets not change all templates and much rather calculate it via a default variable and convert maas_check_timeout into maas_check_timeout * 1000 if we detects the input is seconds as example (ie the timeout is only 3 digits long etc)

Comment (npawelek) [reply playbooks/templates/rax-maas/ceph_cluster_stats.yaml.j2]: The problem is that the check-level still utilizes seconds, but doesn't actually do anything for `agent.plugin` checks. These check types require a millisecond declaration:

https://developer.rackspace.com/docs/rackspace-monitoring/v1/tech-ref-info/check-type-reference/#agent-plugin

File: playbooks/files/rax-maas/plugins/swift-recon.py (hunk 1 above)
Comment (npawelek) [reply playbooks/files/rax-maas/plugins/swift-recon.py]: done

**Comment by npawelek** [comment]
recheck_cit_xenial_pike_deploy

**Comment by npawelek** [comment]
recheck_cit_trusty_mitaka_deploy

**Comment by npawelek** [comment]
recheck_cit_trusty_mitaka_deploy

**Pull request merged**